openwisp_ipam/base/models.py

```diff
@@ -188,6 +188,10 @@ def import_csv(self, file):
             reader = iter(row)
         else:
             reader = csv.reader(StringIO(file.read().decode('utf-8')), delimiter=',')
+        return reader
```

Comment:
I reverted the changes I initially made in this file, but I also changed some logic to make this more flexible. Using `next(reader)` isn't very flexible, we should have read the file as a list and iterated over that instead, but this should do the job for now.